src/Lib/CakeboxInfo.php

```diff
@@ -1147,24 +1112,98 @@ public function getCakeboxCliLog()
     }
 
     /**
-     * Get latest commit header from Github api.
+     * Get notifications.
+     *
+     * @return array Rich hash with notifications or empty array.
+     */
+    public function getNotifications()
+    {
+        $result = [];
+
+        // set update notification for cakebox-console project
+        $cakeboxConsoleUpdate = $this->_getCakeboxConsoleUpdateNotification();
+        if ($cakeboxConsoleUpdate) {
```

Comment:
i wuold prefer to return []; if condition is not met and continue the happy path down